src/main/resources/tsv-schema.json

```diff
@@ -33,11 +33,13 @@
   { "name": "raw sequence data processing method", "valueType": "string" },
   { "name": "dehosting method", "valueType": "string" },
   { "name": "consensus sequence software name", "valueType": "string" },
-  { "name": "consensus sequence software version", "valueType": "number" },
+  { "name": "consensus sequence software version", "valueType": "string" },
```

Comment:
Bug was discovered when this was changed to string. The previous approach was converting string that are numbers to numbers, so setting this to string wasn't working.